ddl/db_integration_test.go

```diff
@@ -2621,6 +2622,69 @@ func (s *testSerialDBSuite1) TestAutoIncrementTableOption(c *C) {
 	tk.MustQuery("select * from t;").Check(testkit.Rows("12345678901234567890"))
 }
 
+func (s *testIntegrationSuite3) TestAutoIncrementForce(c *C) {
```

Comment:
Please add the test case for:
- Rebase for `unsigned` columns.
- `ALTER TABLE ... AUTO_INCREMENT = 0` for different cases.